packages/features/auth/lib/next-auth-options.ts

```diff
@@ -54,6 +56,35 @@ export const checkIfUserBelongsToActiveTeam = <T extends UserTeams>(user: T) =>
     return metadata.success && metadata.data?.subscriptionId;
   });
 
+const checkIfUserShouldBelongToOrg = async (
+  email: string
```

Comment:
Email can't be null, why is there an if statement to check if email is null below?

Reply:
Resolved, thanks